moj-docker-deploy/tag-ebs-volumes.sls

```diff
@@ -0,0 +1,29 @@
+tag-ebs-volumes:
+  file.managed:
+    - source: salt://moj-docker-deploy/files/ebs-tag.py
+    - name: /usr/local/bin/ebs-tag.py
+    - user: root
+    - group: root
+    - mode: '0755'
+    - require:
+      - pip: requests
+      - pip: boto
+  cmd.run:
+    - unless: /usr/local/bin/ebs-tag.py query Env:{{grains['Env']}} Apps:{{grains['Apps']}}
+    - name: /usr/local/bin/ebs-tag.py ensure Env:{{grains['Env']}} Apps:{{grains['Apps']}}
+    - cwd: /
+    - require:
+      - file: /usr/local/bin/ebs-tag.py
+
+python-pip:
+  pkg.installed
```

Comment:
Due to an ubuntu (bug 1306991)[https://bugs.launchpad.net/ubuntu/+source/python-pip/+bug/1306991] bug we're now using a "hack" to install pip. See https://github.com/ministryofjustice/python-formula/commit/00e6b0421cfe36b3d884d9bfb5d1072c4f9de82e

That results in the box not having installed pip as a package, but the pip command is available.